cmd/run.go

```diff
@@ -408,6 +409,24 @@ func setupExecutors(s *scheduler.Scheduler, config *config.Config, conn *connect
 		}
 	}
 
+	if s.WillRunTaskOfType(pipeline.AssetTypeMsSQLQuery) || estimateCustomCheckType == pipeline.AssetTypeMsSQLQuery {
+		msOperator := mssql.NewBasicOperator(conn, wholeFileExtractor, mssql.NewMaterializer())
+
+		msCheckRunner := mssql.NewColumnCheckOperator(conn)
+
+		msCustomCheckRunner := ansisql.NewCustomCheckOperator(conn)
+
+		mainExecutors[pipeline.AssetTypeMsSQLQuery][scheduler.TaskInstanceTypeMain] = msOperator
+		mainExecutors[pipeline.AssetTypeMsSQLQuery][scheduler.TaskInstanceTypeColumnCheck] = msCheckRunner
+		mainExecutors[pipeline.AssetTypeMsSQLQuery][scheduler.TaskInstanceTypeCustomCheck] = msCustomCheckRunner
+
+		// we set the Python runners to run the checks on MsSQL
+		if estimateCustomCheckType == pipeline.AssetTypeMsSQLQuery {
```

Comment:
don't forget to add this to the possible alternatives to the estimator a few lines above